### PR TITLE
doc: mar-2022 OpenSSL security pre-release announcement

### PR DESCRIPTION
**Pull request by joesepi**
This was shared in private org/repo and approved there

### CODE REVIEW

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The OpenSSL project
+[announced](https://mta.openssl.org/pipermail/openssl-announce/2022-March/000216.html)
+this week that they will be releasing versions 3.0.2 and 1.1.1n on the 15th of
+March 2022 between 1300-1700 UTC. The releases will fix two security defects that are
+labelled as "HIGH" severity under their
+[security policy](https://www.openssl.org/policies/secpolicy.html).
+
+Node.js v12.x, v14.x and v16.x use OpenSSL v1.1.1 and Node.js v17.x uses OpenSSL
+v3, therefore all active release lines are impacted by this update.
```
Comment (Trott) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md]:
```suggestion
v3. Therefore all active release lines are impacted by this update.
```

**Review by Trott — APPROVED** [review]

File: locale/en/blog/vulnerability/mar-2022-security-releases.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+title: OpenSSL security releases may require Node.js security releases
+slug: openssl-and-high-severity-fixes-mar-2022
+layout: blog-post.hbs
+author: Joe Sepi
+---
+
+### Summary
+
+The Node.js project may be releasing new versions across all of its supported
+release lines late next week to incorporate upstream patches from OpenSSL.
```
Comment (richardlau) [inline locale/en/blog/vulnerability/mar-2022-security-releases.md:13]: I think we were planning for late this week? (It was "next week" last week when we were having the discussions.)

Comment (Trott) [reply locale/en/blog/vulnerability/mar-2022-security-releases.md:13]: I'll open a PR to fix that. Sorry about the auto-merge.

**Comment by nschonni** [comment]
Should this get added using the banner?

**Comment by mhdawson** [comment]
@nschonni yes, @joesepi I should have mentioned that part we add both the blog and point to it in the banner.

**Pull request merged**